flink-core/src/main/java/org/apache/flink/configuration/HistoryServerOptions.java

```diff
@@ -99,6 +99,11 @@
 			.withDescription("Enable HTTPs access to the HistoryServer web frontend. This is applicable only when the" +
 				" global SSL flag security.ssl.enabled is set to true.");
 
+	public static final ConfigOption<Integer> HISTORY_SERVER_MAX_SIZE =
+		key("historyserver.max.history.size")
```

Comment:
Maybe
```suggestion
		key("historyserver.retainedJobs")
```

Reply:
Or `historyserver.history.max-size`

Reply:
Thanks Till, I agree, the current name does not fit in the already established hierarchical structure. I'd actually build upon your proposal and suggest the name `historyserver.archive.retained-jobs`, which seems to be even more consistent with the existing parameters, given that the current behaviour works on the same abstraction level as the behaviour controlled by `historyserver.archive.clean-expired-jobs`. What do you think?

Reply:
Sounds good to me.